onnx/defs/tensor/defs.cc

```diff
@@ -1319,4 +1319,22 @@ ONNX_OPERATOR_SET_SCHEMA(
             }
           }
         }));
+
+ONNX_OPERATOR_SET_SCHEMA(
+    IsNaN,
+    9,
+    OpSchema()
+        .SetDoc(R"DOC(Returns which elements of the input are NaN.)DOC")
```

Comment:
please add some test cases for this operator. Refer to https://github.com/onnx/onnx/tree/master/onnx/backend/test/case/node for examples.

Reply:
i had one. forgot to check it in.